install_files/securedrop-grsec/DEBIAN/postinst

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+# postinst script for securedrop-grsec
+#
+# see: dh_installdeb(1)
+set -e
+set -x
+# summary of how this script can be called:
+#        * <postinst> `configure' <most-recently-configured-version>
+#        * <old-postinst> `abort-upgrade' <new version>
+#        * <conflictor's-postinst> `abort-remove' `in-favour' <package>
+#          <new-version>
+#        * <postinst> `abort-remove'
+#        * <deconfigured's-postinst> `abort-deconfigure' `in-favour'
+#          <failed-install-package> <version> `removing'
+#          <conflicting-package> <version>
+# for details, see http://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+case "$1" in
+    configure)
+
+    # If the SecureDrop instance is running a rolled-back kernel, this will
+    # preserve the kernel boot priority specfied in its ordinal form by its
+    # canonical form. (In this specific case, 1>2 as GRUB_DEFAULT in
+    # /etc/default/grub indicates the instance is running 3.14.79-grsec)
+    # Otherwise, we want to run the latest 4.4 series kernel.
+    if grep -qE '^GRUB_DEFAULT=\"1>2\"' /etc/default/grub; then
+        sed -i 's/^\(GRUB_DEFAULT=\)"1>2"/\1\"Advanced options for Ubuntu>Ubuntu, with Linux 3.14.79-grsec\"/' /etc/default/grub
```

Comment:
There's a case where this regex won't catch a valid configuration: if single quotes instead of double quotes were used to encase the value. (Haven't tested, but perhaps no quotes at all should be supported, as well.) We should expand the regex to handle a set of possible quote characters, to make sure we're preserving rolled back instances.